marketplaceItems/drawApp/fingerDraw_app.js

```diff
@@ -81,13 +101,28 @@ Tablet, Vec3, Window */
             grab: { grabbable: false },
             collisionless: true
         }, 'avatar');
+        paintSphereMaterial = Entities.addEntity({
+            type: "Material",
+            name: "Draw App Material",
+            materialURL: "materialData",
+            priority: 1,
+            parentID: paintSphere,
+            materialData: JSON.stringify({
+                materials: {
+                    albedo: HIFI_COLORS[randomHiFiColorIndex],
+                    emissiveMap: HIFI_COLORS_URLS[randomHiFiColorIndex]
+                }
+            })
+        },true);
```

Comment:
please fix spacing and replace second argument with 'avatar' to be in line with new standards